src/components/cards/card-property.tsx

```diff
@@ -13,43 +8,63 @@ import { Price } from '../../styles/typography/prices';
 import { Caption } from '../../styles/typography/caption';
 import { useState } from 'react';
 import { CardContainer } from './sub-components/card-container';
+import { formatPrice } from '../../utils/formatPrice';
 
 export interface CardPropertyProps {
-  image: string;
-  title: string;
-  description: string;
+  image: string[];
+  streetNumber: string;
+  district: string;
+  city: string;
+  state: string;
   price: number;
   beds: number;
   bath: number;
   dimension: string;
 }
 
-export const CardProperty = ({ image, title, description, price, beds, bath, dimension }: CardPropertyProps) => {
+export const CardProperty = ({
+  image,
+  price,
+  beds,
+  bath,
+  dimension,
+  streetNumber,
+  district,
+  city,
+  state,
+}: CardPropertyProps) => {
   const [isHearted, setIsHearted] = useState(false);
 
   const handleHeartClick = () => {
     setIsHearted(!isHearted);
   };
 
+  const propertyImage = image.length > 0 ? image[0] : '';
+
   return (
     <CardContainer>
       <div className="property-image-container">
-        <img src={image} alt={title} />
+        <img src={propertyImage} alt="property-imovel" />
       </div>
 
       <div className="price-container">
         <Price color="CTA" size="medium">
           {formatPrice(price)}
         </Price>
         <FontAwesomeIcon
+          color="#8E8E8E"
           icon={isHearted ? faHeartSolid : faHeartRegular}
           className={isHearted ? 'hearted' : ''}
           onClick={handleHeartClick}
         />
       </div>
       <div className="description-container">
-        <Body2 weight="bold"> {title}</Body2>
-        <Caption>{description}</Caption>
+        <Body2 weight="bold">
+          {city}, {state}
+        </Body2>
```

Comment:
nesse caso, é mais seguro usar template string, já que os espaços são importantes

```suggestion
        <Body2 weight="bold">
          {`${city}, ${state}`}
        </Body2>
```